### PR TITLE
Final reviews

### PR DESCRIPTION
**Pull request by ayodelevm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,3 +10,6 @@ script:
   - gulp run-tests
   - npm run test
   - npm run coveralls
+
+notifications:
+  webhooks: https://coveralls.io/webhook?repo_token=o4Q54byGsbo2r7laPPTHtwjEIQDr5BaXK
```
Comment (vynessa) [inline .travis.yml:15]: Isn't this token meant to hidden? If no, why?

File: src/inverted-index.js [hunk 2]
```diff
@@ -221,7 +247,8 @@ export default class InvertedIndex {
  * @return {String} - returns sanitized data
  */
   static sanitizeData(data) {
-    return data.toLowerCase().replace(/[^a-z A-Z 0-9]+/g, '').split(/[\s]\s*/);
+    return data.toLowerCase()
+      .replace(/[^a-z A-Z 0-9]+/g, '').split(/[\s]\s*/);
   }
 
 }
```
Comment (vynessa) [inline src/inverted-index.js:254]: Your code is very readable and it covers a lot of edge cases. Good job.

File: .travis.yml (hunk 1 above)
Comment (ayodelevm) [reply .travis.yml:15]: I'm not sure about whether or not it's meant to be hidden... I added this line when my coveralls stopped building, to allow for parallel builds, based on a guide from coveralls... You can check https://coveralls.zendesk.com/hc/en-us/articles/203484329

File: src/inverted-index.js (hunk 2 above)
Comment (ayodelevm) [reply src/inverted-index.js:254]: Thanks!

File: README.md [hunk 3]
```diff
@@ -1,3 +1,58 @@
...
+## Tests
+*  The tests have been written using Jasmine-node **[TestCase](https://github.com/mhevery/jasmine-node)** and Supertest **[TestCase](https://www.npmjs.com/package/supertest)** class.
+*  They are run using the **`coverage`** tool in order to generate test coverage reports.
+*  To run the tests, navigate to the project's folder and open
+*  Issue the following command on terminal.
+  *  `gulp run-tests`
+* To view coverage, issue the following command on the terminal.
+  * `gulp coverage`
+*  If the tests are successful, they will complete without failures or errors.
+
```
Comment (EmaEvidence) [inline README.md:58]: I think you should add the following to your README.md
 - what your app can't do
 - link to Heroku
 - How to test the API Using POSTMAN.
 - An Example of your file structure.
 - Keys to use when using POSTMAN.

Comment (ayodelevm) [reply README.md:58]: Alright. Thanks!

**Pull request merged**